src/index.html

```diff
@@ -11,7 +11,186 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+
+        <label>
+          Surname:
+
+          <input
+            type="text"
+            name="userSurname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__field">
+          Name:
+
+          <input
+            type="text"
+            name="userName"
+            autocomplete="off"
+          >
+        </label>
+
+
+        <label class="form__field">
+          How old are You?
+
+          <input
+            type="number"
+            name="userAge"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form__field">
+          Full date of birth:
+
+          <input
+            type="date"
+            name="dateOfBirth"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+
+          <input
+            type="checkbox"
+            name="acceptAgreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>
+          Registration
+        </legend>
+
+        <label class="form__fieldShort">
+          E-mail:
+
+          <input
+            type="email"
+            name="userEmail"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label>
+          Password:
+
+          <input
+            type="password"
+            name="userPassword"
+            minlength="8"
+            maxlength="24"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <label>
+          Do you love cats?
+
+          <label>
+            <input
+              type="radio"
```

Comment:
Please check this. You should provide the same name for radio buttons and provide value attributes(yes, no) respectively
![image](https://user-images.githubusercontent.com/54936044/229271269-9a9f5004-5cdb-4d91-883d-e564b807f4d4.png)